test/demo-contracts/UniswapV2SwapRenamed.WETH.sol

```diff
@@ -20,6 +20,14 @@ contract wETHMock {
         return dec;
     }
 
+    function name() public returns (string memory) {
+        return "WETH Coin";
```

Comment:
Name as `Wrapped Ether`, refer to official contract `name`: https://etherscan.io/token/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2#readContract#F1

Reply:
Thank you!